src/views/product-downloads-view/server-helpers.ts

```diff
@@ -41,7 +41,7 @@ export const generatePackageManagers = async ({
 			const { commands } = packageManager
 			const installCodeHtml = await highlightString(
 				generateCodeSnippetFromCommands(commands),
-				'shell-session'
+				'bash'
```

Comment:
Without `$` prefixes, Prisma doesn't apply syntax highlighting. Switching to `bash` gets syntax highlighting working again.